<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerAmplifierTester</class>
 <widget class="QMainWindow" name="PowerAmplifierTester">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>PowerAmplifierTester</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>501</width>
      <height>108</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>231</width>
      <height>39</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="ssweep_pushButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>SSweep</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="spng_pushButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>save PNG</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sdat_pushButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>save dat</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="saveDir_lineEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>D:\\PA_mes\\</string>
    </property>
   </widget>
   <widget class="QPushButton" name="alltest_pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Сделать хорошо!</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>140</y>
      <width>252</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="sA_label">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Ip spec an</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="sA_pushButton">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Init SA</string>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ipSA_lineEdit">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>TCPIP::192.168.29.2::INSTR</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="Gen_label">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Ip generator</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="genSetUpsA_pushButton">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>GenSetUp</string>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ipGen_lineEdit">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>TCPIP::192.168.29.1::INSTR</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>21</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QComboBox" name="modType_comboBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="maxVisibleItems">
        <number>2</number>
       </property>
       <property name="maxCount">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>APSK32 9/10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QPSK</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="symLen_label">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Sym length</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="symLen_lineEdit">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>32768</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="refLevellabel">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Ref level</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="alpha_label">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>RRC alpha</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="sumRate_lineEdit">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>1e6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="refLevel_lineEdit">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="alpha_lineEdit">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>0.22</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="modType_label">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Mod type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="cf_lineEdit">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>1e9</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="cf_label">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Central FREQ</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="sumRate_label">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Sum rate</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>saveDir_lineEdit</zorder>
   <zorder>alltest_pushButton</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu12">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu12"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>sumRate_lineEdit</tabstop>
  <tabstop>symLen_lineEdit</tabstop>
  <tabstop>cf_lineEdit</tabstop>
  <tabstop>modType_comboBox</tabstop>
  <tabstop>sA_pushButton</tabstop>
  <tabstop>refLevel_lineEdit</tabstop>
  <tabstop>alpha_lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>PowerAmplifierTester</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
